ui.frontend/src/view/FormField.js

```diff
@@ -77,9 +77,10 @@ class FormField {
     setActive() {
         if (!this.isActive()) {
             this.element.setAttribute(Constants.DATA_ATTRIBUTE_ACTIVE, true);
-        }
-        if (this.parentView && this.parentView.setActive) {
-            this.parentView.setActive();
+            if (this.parentView) {
+                this.parentView._model.activeChild = this._model; // updating the activeChild of the model when a field is focused in view
```

Comment:
this should be done via runtime. any reason to do it again?
typically,  whenever a rule is invoked to set focus, runtime will update the active child and then view will receive that event to update the markup.

Reply:
@nit23uec 
If a user clicks/focuses on a field in view that was not getting set in model. 
For next, prev button we need the navigation to move w.r.t the current active field.